data/json/furniture_and_terrain/appliances.json

```diff
@@ -32,6 +32,40 @@
     ],
     "damage_reduction": { "all": 32 }
   },
+  {
+    "id": "ap_oven",
+    "name": { "str": "oven" },
+    "description": "A standard electric convection-based oven, commonly used for heating and cooking food.",
+    "symbol": "#",
+    "color": "dark_gray",
+    "broken_color": "light_cyan",
+    "broken_symbol": "x",
+    "categories": [ "utility" ],
```

Comment:
```suggestion
```
I think this doesn't apply for appliances either but I haven't tested removing it